src/filters/AIHelp.ts

```diff
@@ -0,0 +1,58 @@
+import { Filter } from "./filter";
+import { Message, MessageEmbed } from "discord.js";
+import {Wit, log} from 'node-wit';
+import * as config from "../config.json";
+export class AIHelp extends Filter {
+    constructor() {
+        super(9);
+    }
+    async run(message: Message): Promise<boolean> {
+        if (message.author.bot) {
+            return true;
+        }
+        else {
+            const client = new Wit({accessToken: config.witToken});
+            client.message(message.content, {})
+            .then((data) => {
+                let intent = data.intents[0].name; 
+                if(data.intents[0].confidence > 0.94){
+                    console.log('Yay, got Wit.ai response: ' + JSON.stringify(data));
+                    if(intent == "wherekeys"){
```

Comment:
same as above. And it will look better if you will use enum for 'wherekeys', 'drivers' and etc. And istead of if use switch-case. The code will look a way better